api/controllers/AuthController.ts

```diff
@@ -18,21 +19,19 @@ import { authActionMetadata } from '../../utils/AuthActionMetadata';
 import { OptionalUserAuthentication } from '../middleware/UserAuthentication';
 import { AuthenticatedUser } from '../decorators/AuthenticatedUser';
 import { UserModel } from '../../models/UserModel';
+import { ValidEmail, ValidAccessCode } from '../validators/GenericRequests';
 
 @JsonController('/auth')
 export class AuthController {
+  @Inject()
   private userAccountService: UserAccountService;
 
+  @Inject()
   private userAuthService: UserAuthService;
 
+  @Inject()
   private emailService: EmailService;
 
```

Comment:
Why's the `@Inject()` decorator added back? Are these merged changes from earlier or are they required for the controller now? 

I think we removed those (and added the constructor in below lines) back in an earlier PR for getting testing setup, so this change would break most of the test suites

Reply:
Oooop, I must have accidently kept them when merging, my b